src/main/kotlin/no/nav/familie/ef/søknad/mapper/kontrakt/BarnepassMapper.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.ef.søknad.mapper.kontrakt
+
+import no.nav.familie.ef.søknad.mapper.tilDesimaltall
+import no.nav.familie.ef.søknad.mapper.tilLocalDate
+import no.nav.familie.ef.søknad.mapper.tilSøknadsfelt
+import no.nav.familie.kontrakter.ef.søknad.Barnepass
+import no.nav.familie.kontrakter.ef.søknad.BarnepassOrdning
+import no.nav.familie.kontrakter.ef.søknad.Periode
+import no.nav.familie.kontrakter.ef.søknad.Søknadsfelt
+import no.nav.familie.ef.søknad.api.dto.søknadsdialog.Barnepass as BarnepassDto
+
+object BarnepassMapper {
+
+    fun tilBarnepass(barnepass: BarnepassDto): Barnepass {
+        return Barnepass(årsakBarnepass = barnepass.årsakBarnepass?.tilSøknadsfelt(),
+                         barnepassordninger = Søknadsfelt("", barnepass.barnepassordninger.map { // TODO
```

Comment:
@charliemidtlyng @miichu - label her?

Reply:
Ser ut som alle labels på lister er hardkodet i API-et fra før.

Reply:
Har jeg misforstått noe her?

```
IBarnepass {
årsakBarnepass: ISpørsmålFelt;
}
```

ISpørsmålFelt skal ha spørsmålid, svarid, label og verdi. 
Er dette fremdeles et frontendproblem eller er det api som ikke har tatt i bruk riktige felter? 🤔

Reply:
`barnepassordninger = Søknadsfelt("",` <--- Som du ser så har ikke listen barnepassordninger en egen label - det har vi ikke for overgangsstønad heller ettersom vi sender en liste fra frontend.